### ISSUE TITLE
Documentation missing typography.cjs for tailwind.config.cjs

### ISSUE THREAD

**Issue by figuerom16** [Brain-Bones/skeleton#273]
Issue: Typography doesn't work without typography.cjs in tailwind.config.cjs for https://skeleton.brainandbonesllc.com/tailwind/typography

Documentation update: https://skeleton.brainandbonesllc.com/guides/tailwind

Section:
```
module.exports = {
    // ...
    plugins: [
        // Keep any existing plugins present and append the following:
        require("@brainandbones/skeleton/tailwind/theme.cjs")
    ]
}
```

Needs typography.cjs:
```
module.exports = {
    // ...
    plugins: [
        // Keep any existing plugins present and append the following:
        require("@brainandbones/skeleton/tailwind/theme.cjs")
        require("@brainandbones/skeleton/tailwind/typography.cjs")
    ]
}
```

**Comment by endigo9740** [comment]
Hey @figuerom16, actually that's not quite right. The typography plugin was merely a test. It may even be removed in the next release. If your typography isn't working it's likely an issue with your Tailwind directives, especially if they are included twice in the project.

We ended up debugging this very issue before release yesterday. This is why we added the big yellow message here:
https://skeleton.brainandbonesllc.com/guides/styling

For us, the issue was that we imported `all.css`, but we forgot to remove the Tailwind directives in our global stylesheet that Svelte Add had inserted for us. Very easy thing to miss! If you're having this issue try this and see if it helps.
